docs/bigquery-usage.rst

```diff
@@ -0,0 +1,49 @@
+Using the API
+=============
+
+Authorization / Configuration
+-----------------------------
+
+- Use :class:`Client <gcloud.bigquery.client.Client>` objects to configure
+  your applications.
+
+- :class:`Client <gcloud.bigquery.client.Client>` objects hold both a ``project``
+  and an authenticated connection to the PubSub service.
+
+- The authentication credentials can be implicitly determined from the
+  environment or directly via
+  :meth:`from_service_account_json <gcloud.bigquery.client.Client.from_service_account_json>`
+  and
+  :meth:`from_service_account_p12 <gcloud.bigquery.client.Client.from_service_account_p12>`.
+
+- After setting ``GOOGLE_APPLICATION_CREDENTIALS`` and ``GCLOUD_PROJECT``
+  environment variables, create an instance of
+  :class:`Client <gcloud.bigquery.client.Client>`.
+
+  .. doctest::
```

Comment:
If we ever get around to addressing #212, the `doctest` sections should be only the relevant example (rather than setup / teardown code, which goes into `testsetup` and 'testcleanup').